stock_quant_cost_segmentation/model/stock.py

```diff
@@ -134,11 +134,8 @@ def _compute_segmentation(self):
               "`material cost`, `subcontracting cost`, `landed cost` "
               "& `production cost`"))
 
-    @api.v7
-    def initializing_quant_segmentation(
-            self, cr, uid, ids=None, context=None):
-
-        cr.execute('''
+    def initializing_quant_segmentation(self):
```

Comment:
Line: 
https://github.com/Vauxoo/addons-vauxoo/blob/8.0/stock_quant_cost_segmentation/data/data.xml#L21

Need to be changed to new api signature:
`env['stock.quant'].initializing_quant_segmentation()`